components/layout/Header.js

```diff
@@ -89,96 +92,97 @@ function Header() {
-          >
-            Sign Up
-          </button>
-        </div>
-      </div>
-
-      {/* For Mobile Navigation */}
-      <div className="sm:tw-hidden tw-flex tw-relative" ref={dropdownRef}>
-        <div className="tw-flex">
-          <div
-            className="tw-text-3xl tw-font-light tw-mx-2 cursor-pointer tw-text-[#845ec2]"
-            onClick={() => {
-              setToggleDropdown((prev) => !prev);
-            }}
-          >
-            <ion-icon name={toggleDropdown ? "close" : "menu"}></ion-icon>
+            <button
+              type="button"
+              onClick=""
+              className="tw-text-white tw-mb-8 tw-bg-[#845ec2] hover:tw-scale-[0.99] active:tw-scale-[1.01] active:tw-shadow-none tw-font-medium tw-rounded-xl tw-text-md tw-px-3 tw-py-2 tw-text-center tw-mr-3 sm:tw-mb-0 md:tw-mr-5 "
+            >
+              Sign Up
+            </button>
           </div>
-          {toggleDropdown && (
-            <div className="tw-absolute tw-right-0 tw-top-full tw-mt-0 tw-w-full tw-p-6 tw-rounded-lg tw-bg-white tw-min-w-[100vw] tw-flex tw-flex-col tw-gap-5 tw-justify-center tw-items-center tw-text-sm tw-font-semibold tw-pb-5">
-              <Link
-                href="/"
-                className="tw-text-xl tw-p-2 tw-font-inter tw-text-gray-500  hover:tw-text-gray-500 tw-font-medium"
-              >
-                Home
-              </Link>
-              <Link
-                href="/mentors"
-                className="tw-text-xl tw-p-2 tw-font-inter tw-text-gray-500   hover:tw-text-gray-500 tw-font-medium"
-              >
-                Mentor
-              </Link>
-              <Link
-                href="/contact"
-                className="tw-text-xl tw-p-2 tw-font-inter tw-text-gray-500  hover:tw-text-gray-500 tw-font-medium"
-              >
-                Contact
-              </Link>
-
-              <Link
-                href="/"
-                className="tw-text-xl tw-p-2 tw-font-inter tw-text-gray-500  hover:tw-text-gray-500 tw-font-medium"
-              >
-                Sign In
-              </Link>
-              <SunIcon className="tw-h-10 tw-bg-white tw-rounded-2xl tw-px-0.5 tw-w-9 tw-text-yellow-400 tw-cursor-pointer" />
-              <button
-                type="button"
-                onClick=""
-                className=" tw-w-full tw-rounded-lg tw-border btn:hover btn:active tw-bg-[#845ec2] hover:tw-scale-[0.99] active:tw-scale-[1.01] active:tw-shadow-none tw-py-3 tw-text-white tw-transition-all tw-text-center tw-text-md tw-font-inter tw-flex tw-items-center tw-justify-center"
-              >
-                Signup
-              </button>
-
-              {/* <DropdownCard
+        </div>
+
+        {/* For Mobile Navigation */}
+        <div className="sm:tw-hidden tw-flex tw-relative" ref={dropdownRef}>
```

Comment:
need to be changed here, If you clearly see when the dropdown opens it does not converge the full width of the device.